featurebyte/__init__.py

```diff
@@ -152,6 +159,8 @@ def get_active_profile() -> Profile:
                     api_token=api_token,
                     make_default=True,
                 )
+    except (ModuleNotFoundError, ImportError, ValueError):
```

Comment:
Where is ValueError expected to be raised?

Reply:
it's triggered if databricks SDK is installed and google credentials env variable is also set
```
ValueError: default auth: cannot configure default credentials, please check https://docs.databricks.com/en/dev-tools/auth.html#databricks-client-unified-authentication to configure credentials for your preferred authentication method. Config: google_credentials=***. Env: GOOGLE_CREDENTIALS
```